feathr-impl/src/main/scala/com/linkedin/feathr/offline/client/FeathrClient.scala

```diff
@@ -95,8 +95,9 @@ class FeathrClient private[offline] (sparkSession: SparkSession, featureGroups:
    *         type of exception.
    */
   def doJoinObsAndFeaturesWithSuppressedExceptions(joinConfig: FeatureJoinConfig, obsData: SparkFeaturizedDataset,
```

Comment:
Missing annotation for @InternalApi and we don't need the function just about this now right?